content/applications/services/planning.rst

```diff
@@ -0,0 +1,323 @@
+:show-content:
+:show-toc:
+
```

Comment:
This is needed only if you have children pages. So it will probably be needed at some point, but currently it is not needed. To have a clean code and commit, it's best to include this kind of code only when it's needed, that is when you'll create a new PR with the children pages.

Reply:
Done <3